App/src/pages/AddUserToGroup.tsx

```diff
@@ -0,0 +1,125 @@
+import { useEffect, useState } from "react";
+import logo from "../assets/logo.png";
+import { Container, Stack, Heading, Box, FormControl, Input, FormLabel, HStack } from "@chakra-ui/react";
+import { useNavigate } from "react-router-dom";
+import Button from "../components/Button";
+import { AddUserToGroup, GetGroupByID, GroupDTO, GroupResponseDTO } from "../services/group";
+import { useLocation } from "react-router-dom";
+import { BaseRoutes } from "../constants";
+
+export default function AddingUserToGroup() {
+    const userList = [
+        { id: "1", email: "User1@gmail.com", firstName: "User", lastName: "1" },
+        { id: "2", email: "Dave@gmail.com", firstName: "Dave", lastName: "1" },
+        { id: "3", email: "Joe@gmail.com", firstName: "Joe", lastName: "1" },
+        { id: "4", email: "Billy@gmail.com", firstName: "Billy", lastName: "1" },
+        { id: "5", email: "Benjamen@gmail.com", firstName: "Benjamen", lastName: "1" },
+    ];
+    const navigate = useNavigate();
+    const location = useLocation();
+    const [filteredList, setFilteredList] = useState(userList);
+    const [myGroup, setMyGroup] = useState<GroupDTO | null>(null);
+    const [locationState, setLocationState] = useState({ gid: "", name: "" });
+
+    function getMyGroup(gid: string) {
+        console.log("Grabbing group info");
+        GetGroupByID(gid)
+            .then((res: GroupResponseDTO) => {
+                setMyGroup(res.data);
+                console.log("this is the name" + res.data.name);
```

Comment:
I think these debugging console logs can be removed

Reply:
fair, I'll remove them